deployment/uwsgi/uwsgi.ini

```diff
@@ -0,0 +1,29 @@
+[uwsgi]
+protocol = uwsgi
+socket = /var/run/gen3/uwsgi.sock
+buffer-size = 32768
+chmod-socket = 666
+master = true
+processes = 2
```

Comment:
A failing database schema migration may prevent one of the worker processes from importing the WSGI application correctly, leading to a situation with 1 functional process and 1 broken one. I'm still thinking about how to properly handle that.